docs/source/api/v3/servercheck.rst

```diff
@@ -31,7 +31,34 @@ Fetches identifying and meta information as well as "check" values regarding all
 
 Request Structure
 -----------------
-No parameters available.
+.. table:: Request Query Parameters
+
+	+-----------+------------------------------------------------------------------------------------+
+	| Name      | Description                                                                        |
+	+===========+====================================================================================+
+	| id        | Return only :term:`Servers` with this integral, unique identifier (id)             |
+	+-----------+------------------------------------------------------------------------------------+
+	| name      | Return only :term:`Servers` with this host_name                                    |
+	+-----------+------------------------------------------------------------------------------------+
+
+.. code-block:: http
```

Comment:
I'm not sure exactly why, but the docs build shows 'Error in "code-block" directive'